configs/retinaface/retinaface_r152_fpn_ssh_widerface.py

```diff
@@ -0,0 +1,184 @@
+_base_ = [
+    '../_base_/default_runtime.py',
+]
+
+dataset_type = 'WIDERFaceDataset'
+data_root = 'data/widerface_voc/'
+img_norm_cfg = dict(mean=[123.675, 116.28, 103.53], std=[1, 1, 1], to_rgb=True)
+data = dict(
+    samples_per_gpu=8,
+    workers_per_gpu=8,
+    train=dict(
+        type=dataset_type,
+        ann_file=data_root + 'WIDER_train/train.txt',
+        img_prefix=data_root + 'WIDER_train/',
+        min_size=1,
+        offset=0,
+        pipeline=[
+            dict(type='LoadImageFromFile', to_float32=True),
+            dict(type='LoadAnnotations', with_bbox=True),
+            dict(type='RandomSquareCrop',
```

Comment:
Is it possible to use `RandomCenterCropPad` or `MinIoURandomCrop` or `RandomCrop` directly